core/src/main/java/com/radixdlt/ledger/StateComputerLedger.java

```diff
@@ -162,7 +161,8 @@ StateComputerPrepareResult prepare(
         List<RawNotarizedTransaction> proposedTransactions,
         RoundDetails roundDetails);
 
-    LedgerProofBundle commit(LedgerExtension ledgerExtension, VertexStoreState vertexStore);
+    LedgerProofBundle commit(
+        LedgerExtension ledgerExtension, Option<byte[]> serializedVertexStoreState);
```

Comment:
Out of interest, why is this `Option<byte[]>` and not `Option<WrappedByteArray>`?

Reply:
no reason at all, I'll change to `Option<WrappedByteArray>`